src/engine/Interfaces/IAudio.ts

```diff
@@ -1,6 +1,16 @@
 import { Promise } from './../Promises';
 
 /**
+ * Defines the points which describe a cubic bezier curve for a sound profile
+ */
+export interface IBezierPoints {
+  point1: number;
+  point2: number;
+  point3: number;
+  point4: number;
+}
```

Comment:
It can. The general definition of a bezier curve is given [here](https://en.wikipedia.org/wiki/B%C3%A9zier_curve#Cubic_B.C3.A9zier_curves). I like the cubic because it allows for up to two changes in direction, and can do everything the quadratic and linear equations can do. Getting much more than 4 bezier points starts to become maybe more confusing than necessary in my opinion, but it can be done